### PR TITLE
Sequence variants

### PR DESCRIPTION
**Pull request by DaniDelHoyo**

### CODE REVIEW

**Review by DaniDelHoyo — COMMENTED** [review]
Correct reviews

File: pwchem/protocols/protocol_multipleSequence_alignment.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            outputFormat = clustaloFormats[formatSelectedClustal]
+            print('Output Format: ', outputFormat)
+            rootProgram = "clustalOmega."
+            if outputFormat == 'ClustalO':
+                output_file = self._getPath(rootProgram + 'aln')
+            else:
+                output_file = self._getPath(rootProgram + clustalFormatsArgs[formatSelectedClustal])
+
+            # Alignment
+            run_alignment = setForAlignment.clustalOmegaAlignSequences(input_file, output_file)
```
[the remark this answers is not in the document]
Comment (eugeUlVe) [reply pwchem/protocols/protocol_multipleSequence_alignment.py:100]: Functions clustalOmegaAlignSequences and muscleAlignmentSequences have been moved to ~/scipion3/scipion-chem/utils/utilsFasta.py

[the remark this answers is not in the document]
Comment (DaniDelHoyo) [reply pwchem/protocols/protocol_multipleSequence_alignment.py:100]: You have to upload the changes for this

File: pwchem/protocols/protocol_variantSequence.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    def generateVariantStep(self):
+        selectedVariant = self.selectVariant.get()
+        inputSequenceVar = self.inputSequenceVariants.get()
+        varDic = inputSequenceVar.getMutationsInLineage()
+        print('List of mutations: ', varDic[selectedVariant])
+        id_sequence = self.inputSequenceVariants.get().getId()
+
+        variantSequence = inputSequenceVar.generateVariantLineage(selectedVariant)
+        variantSequence = Sequence(sequence=variantSequence, name=(id_sequence + '_VariantSequence'))
```
[the remark this answers is not in the document]
Comment (eugeUlVe) [reply pwchem/protocols/protocol_variantSequence.py:102]: inputSequenceVariants is a SequenceVariants object and SequenceVariants has id and name, e.g. SequenceVariants(id=p0dtc2,name=p0dtc2). Could you please give me more details ?

File: pwchem/protocols/protocol_multipleSequence_alignment.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (eugeUlVe) [reply pwchem/protocols/protocol_multipleSequence_alignment.py:100]: Done !

**Pull request merged**